configure-secrets.sh

```diff
@@ -1,5 +1,29 @@
 #!/bin/sh -e
 
+NOCOLOR='\033[0m'
+RED='\033[0;31m'
+GREEN='\033[0;32m'
+BOLD='\033[1m'
+NOBOLD='\033[21m'
+
+usage () {
+cat <<HELP_USAGE
+usage: $0 param1 param[n]...
+
+   Parameters:
+   - param1: The SDK name which needs to have a -Secrets.xcconfig file
+   - param[n]: a list of secrets environment variables to put in secret config file
+
+  Example usage:
+    $0 MiniApp RMA_API_ENDPOINT RAS_PROJECT_SUBSCRIPTION_KEY RAS_PROJECT_IDENTIFIER
```

Comment:
I think this example doesn't match InAppMessaging :)

Reply:
I don't see what's wrong 😚

Reply:
Shouldn't that be `$0 InAppMessaging RIAM_CONFIG_URL...` instead of `$0 MiniApp...`?

Reply:
I was a joke, sorry XD
Sometimes I should precise ^^

Reply:
done